addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -233,6 +238,14 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
                 return false;
             }
 
+            if (this.currentOrder.is_to_ship() && !this.currentOrder.get_client().address) {
```

Comment:
Shouldn't we check if that all the fields in the address are set? Here, if I set only the country on the customer, there's no error, even though I don't have enough information to send the package.

Reply:
Other problem:
- Create a customer without address
- Open the POS
- When I try to validate the order, I get the error saying that I should set an address
- Set the address on the customer from the POS
- Try to validate the order, I still get the error message saying that the address is missing